src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -1093,80 +1160,19 @@ private async Task<WebResponse> SendRequest()
                 throw new InvalidOperationException(SR.net_reqsubmitted);
             }
 
-            var handler = new HttpClientHandler();
             var request = new HttpRequestMessage(new HttpMethod(_originVerb), _requestUri);
 
-            using (var client = new HttpClient(handler))
+            bool disposeRequired = true;
```

Comment:
Nit: the value that's assigned here doesn't really matter... you're only having to do so because the C# compiler can't see that the value is irrelevant.  But as long as you're setting it, `false` is a more accurate value.